gamemodes/terrortown/entities/weapons/weapon_zm_carry.lua

```diff
@@ -65,40 +65,44 @@ SWEP.PrevOwner = nil
 
 -- ConVar syncing
 if SERVER then
-
 	---
 	-- @realm server
-	local cvAllowRagCarry = CreateConVar("ttt_ragdoll_carrying", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvAllowRagCarry =
+		CreateConVar("ttt_ragdoll_carrying", "1", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
```

Comment:
In another file your CreateConVar looked like this:

```lua
local cvAllowRagCarry = CreateConVar(
	"ttt_ragdoll_carrying",
	"1",
	{ FCVAR_NOTIFY, FCVAR_ARCHIVE }
)
```

Or this way, but at least it should be a single style in the whole project